### DISCUSSION TITLE
Conversione impianto a pavimento

### DISCUSSION THREAD

**Discussion by comdav** [indomus/forum#2507] [GENERALE - Discussioni generiche]
Buongiorno a tutti,

sono un novellino, ho solo letto la sezione Nozioni di base e Software del sito.
Non sono riuscito a capire se ci sono delle valvole termostatiche per convertire un impianto di riscaldamento a pavimento.
Il mio ha due collettori come questo nell'immagine.

![collettore](https://github.com/indomus/forum/assets/64091690/df0e701a-5a38-49ea-a858-ea9cf0d8da12)

L'idraulico mi ha detto che ci sono delle valvole che devono essere comandate da una centrale, tramite cavi elettrici, a 220volt o 24volt. Ma non è un sistema domotico.

Mi potete far capire che tipo di valvole ci voglio?
Si possono usare quelle per i termosifoni tipo le Tado: https://indomus.it/componenti/testa-termostatica-intelligente-tado/
Esistono delle versioni apposta, sempre della Tado per esempio.
Oppure devo montare le valvole che mi ha parlato l'idraulico a 220volt e poi le comando con degli attuatori tipo: https://indomus.it/componenti/itead-sonoff-4ch-4ch-pro-r3/

Grazie
Saluti
Davide

**Comment by dante70** [comment]
ciao
le valvole che hai menzionatto per radaiatori non sono adatte ad essere montate nel tuo impianto in quanto appunto nate per modulare portata d'aqua ad un singlo radiatore in funzione della temperatura rilevata dal sensore incorporato
Da quello che hai esposto penso che la soluzione indicata dal tuo idraulico possa essere giusta.......ma attenzione non sono le valvole che devono essere integrabili  in domotica.
mi spiego meglio......tu ora come gestisci il riscaldamento?avrai pure un termostato,o meglio cronotermostato che gestisce magari se non ci sono valvole le pompe di rilancio alle varie zone?E' li che ci dobbiamo concentrare e poi credo senza fare modifiche idrauliche gestirai con semplici rele' integrabili in domotica.
Dicci qualcosa di piu' di tutto il sistema e arriveranno consigli più mirati

**Reply by robibbj61** [reply]
Guarda su AliExpress : concentratore a 8 zone della Beok che è integrabile in HA.

**Comment by comdav** [comment]
Ti ringrazio molto dante70 della risposta.
Hai ragione devo specificare meglio la strutta.

Attualmente ci sono due zone: giorno e notte.
La zona giorno comprende la cucina e la sala. Il cronotermostato è posizionato in sala. Il collettore (nell'immagine precedente) si trova invece nel corridoio, che oltretutto  è nella zona notte ed al di sopra è posizionato il crono termostato della zona notte.
La zona notte comprende la camera da letto il corridoio, i due bagni, la lavanderia e un camera/studio. Il Cronotermostato, come detto, è nel corridoio e il collettore in lavanderia.
C'è anche un altro impianto per gli scaldasalviette con il cronotermostato in un bagno.

Io volevo avere un termometro in ogni stanza e comandare le valvole nei collettori per gestire l'acqua.
Naturalmente ci sono più circuiti in alcune stanza, come la camera da letto, la sala e la cucina. Quindi due o più valvole saranno comandate dallo stesso relè.
Come hai detto tu, c'è una pompa per ogni zona, ma quella è comandata dalla centralina che a sua volta è comandata dal cronotermostato.
Nella mia ignoranza pensavo che si potrebbe sostituire il cronotermostato con un relè, le valvole da altri relè e Home Assistant ricevendo la temperatura dal termometro della stanza comanda la o le relative valvole e il relè dell'exCronotermostato per avviare la pompa.

Se è corretto. che relè devo usare?
I termometri possono andare bene i Tado o quelli della Xaomi?

Grazie ancora
Davide

**Comment by dante70** [comment]
ciao
ma scusa se ho capito bene vorresti per ogni stanza poter aver il controllo della temperatura?
se e' cosi dovresti montare tante valvole quante sono le stanze  e ognuna di queste comandarle con la relativa temperatura e quando nessuna chiama spegnere pompa assolutamente altrimenti la fai fuori
Mi sembra una cosa impegnativa da realizzare.....a parte i costi,gli spazi dove montare le valvole hai valutato poi che l'impianto a pavimento  con l'inerzia che ha non si presta a regolazioni di questo tipo.
aggiungo anche che idraulicamente ci sarebbero da valutare seriamente il discorso bilanciatura dei flussi(come vedi nei collettori ci sono flussometri a questo scopo)......vedi tu

**Comment by cl-7k-01** [comment]
Ciao,
un impianto di riscaldamento a pavimento solitamente è diviso in zone (5 nel tuo caso) che spesso, ma non sempre, corrispondono alle stanze/locali che vanno a servire. A casa mia, per puro esempio, il soggiorno è diviso in 2 zone, omogeneizzando la lunghezza delle serpentine posate.
Secondo me, per fare ciò che ti sei posto, in ogni "mandata"  (dove ci sono i flussometri per capirci) dovrai installare delle tradizionali elettrovalvole per collettore (le più famose sono della marca FAR, 40€ l'una se non erro..) e, credo, una pompa di circolazione perché ho la sensazione che ora, a fare la circolazione sia la pompa della caldaia.
Le elettrovalvole sono comandate da termostati. Alcune volte, in casi come il mio, un termostato può controllare più elettrovalvole, facendole lavorare "sincronizzate" (nel mio caso due zone del soggiorno lavorano insieme).
Le elettrovalvole saranno controllate da un termostato/cronotermostato.
Io cercherei di domotizzare i soli termostati: a cascata avrai domotizzato tutto il tuo impianto.
Il mio suggerimento è di farti sistemare per bene l'impianto di riscaldamento dal tuo idraulico in maniera "tradizionale", con "semplici" elettrovalvole, pompe di circolazione ecc. ecc  per poi domotizzare i "soli" termostati".
Mi collego inoltre a quanto già ti è stato suggerito: il lavoro potrebbe essere "importante" e di difficile realizzazione per costi e spazi.
Seppur vero che l'_inerzia di questi impianti poco si presta a queste regolazioni_, le elettrovalvole ti permetterebbero di "diversificare" il riscaldamento degli ambienti della casa per avere un maggior comfort, uno su tutti, la temperatura della zona notte, ma anche bagno vs. resto della casa o magari qualche locale che, seppur dotato di impianto, non serve venga riscaldato tanto quanto gli altri.
Il mio idraulico, all'epoca mi aveva sconsigliato i cronotermostati che, effettivamente, non hanno molto senso su impianti del genere (abbassare di un grado o due una zona, in intervalli di qualche ora è assolutamente non-sense), mentre è indubbiamente utile la suddivisione in zone, per i motivi di cui sopra.
Concludendo: anche secondo me è il caso di domotizzare i soli termostati una volta fatto un buon impianto suddiviso a zone.
Spero di essere stato utile,
ciao

**Comment by comdav** [comment]
Ciao a tutti,

scusate se rispondo solo adesso, tra lavoro e famiglia mi è stato impossibile.

Sto guardando i prodotti Beok consigliati da robibbj61, grazie.
Devo capire solo la gestione della pompa e del boiler, comunque potrebbe fare al caso mio.

Per rispondere a dante70, l'idraulico ha detto che non è un grande problema il bilanciamento dei flussi.

Per cl-7k-01 non credo sia fattibile collegare le valvole, anche accoppiandole, per la gestione dei circuiti delle stanze dal termostato, ma devo chiedere all'idraulico.
Perché attualmente il termostato, non cronotermostato che hai scritto non ha senso, comanda una centralina che comanda la pompa della zona e una valvola motorizzata per reimmettere l'acqua ancora calda nel circuito.
Quindi ci vorrebbero due relè, uno comanda le valvole della zona desiderata, es. cucina, attivato dal termostato ed uno che attiva al centralina che a sua volta comanda la pompa e la valvola motorizzata.

Grazie a tutti delle risposte.
Davide

**Reply by cl-7k-01** [reply]
Non credo di aver capito bene la tua situazione, ma ribadisco quanto ti è stato detto anche da altri: per domotizzare un impianto di riscaldamento solitamente si interviene sul primo elemento che comanda tutto il resto: il termostato.
Valvole, pompe, consensi in caldaia ecc. ecc. partono tutti alla "richesta" del termostato (o cronotermostato). Domotizzando lui, rendi domotico tutto il resto.
A mio avviso, parlare di domotizzare valvole di flusso, pompe relè o altro non proprio senso, anche perchè, secondo il mio modesto parere il sistema deve essere rindondante: se ti va giù l'hub domotico il riscaldamento dovresti riuscire a farlo partire lo stesso con un semplice comando sul termostato (o cronotermostato).
Poi boh, alzo le mani, non sono un esperto di settore.
